parlai/mturk/core/dev/mturk_manager.py

```diff
@@ -1811,93 +1807,8 @@ def _assert_opts(self):
             )
             raise Exception('Invalid mturk manager options')
 
-    def move_agent_to_task(self, agent, new_conversation_id):
-        agent.set_status(AssignState.STATUS_IN_TASK)
-        self.worker_manager.change_agent_conversation(
-            agent=agent, conversation_id=new_conversation_id, new_agent_id=agent.id
-        )
-        # Remove selected agents from the pool
-        self._remove_from_agent_pool(agent)
-
-    def _on_alive(self, pkt):
-        """Notes a new connection from an agent. In the Static case, this
-        should only be called once per task. If it's called again, the
-        task world will re-send the task details.
+    def _onboard_new_agent(agent):
+        """Override onboarding to go straight to the pool
+        for static stasks
         """
-        shared_utils.print_and_log(logging.DEBUG, 'on_agent_alive: {}'.format(pkt))
```

Comment:
Turns out this was all duplicated just to be able to map `_onboard_new_agent(agent)` to `add_agent_to_pool(agent)`. Silly silly